app/js/views/reminders.jsx

```diff
@@ -104,6 +107,10 @@ export default class Reminders extends React.Component {
       });
   }
 
+  onParsingFailure() {
+    this.speechController.speak(`I didn't understand that. Can you repeat?`);
```

Comment:
Should we immediately start the speech recognition again?  And maybe understand something like "never mind" or "it doesn't matter" to cancel?